draft-ietf-rats-architecture.md

```diff
@@ -1424,30 +1425,57 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
-Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR_v) - time(EG_a) < validity-duration`.
+The distribution of a new handle determines the transition from one epoch to
+the next.  The transition between epochs goes through an "overlap" phase in
+which the previous epoch is still extant while the current epoch is
+established.  The span of the overlap phase is determined solely by the
+transmission delay of the underlying communication channel,
+`time(HR)-time(HT)`.  Once the overlap phase is over, an epoch is fully
+established.
 
-From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "(time(HR'_v)-time(HR_v)) - (time(HR_a)-time(EG_a)) \< validity-duration".
+In order to determine freshness of Evidence or Attestation Results, consumers
+of messages based on handles need to understand whether an epoch is fully
+established or in overlap phase.
 
-Due to the validity-duration of handles, multiple different pieces of Evidence can be generated based on the same handle.
-The resulting granularity (time resolution) of Evidence freshness is typically lower than the resolution of clock-based tickcounters.
+The Handle Distributor, hosted in a dedicated entity or co-located with other
+RATS roles, is trusted by the protocol participants to behave correctly with
+regards to its duty of handle distribution.
 
-The following example illustrates a hypothetical Background-Check Model solution that uses handles and requires a trustworthy time source available to the Handle Distributor role.
+Consumers of Conceptual Messages based on handles must be able to safely trace
+each handle's origin back to the broadcasting Handle Distributor either using
+topological considerations (e.g., co-location), or by cryptographic means.
+
+The example in {{fig-handle-passport}} illustrates a Handle-based interaction
+using the passport topology model.
+
+At `time(HT)`, the Handle Distributor broadcasts handle `H` which starts a new
+epoch `E` for a protocol participant upon reception at `time(HR)`.
+
+The Attester generates Evidence incorporating handle `H` and conveys it to the
+Verifier.
+
+The Verifier appraises that the received handle `H` is "fresh" according to the
+definition provided above, and generates an Attestation Result.  The
+Attestation Result is conveyed to the Attester.
+
+After the transmission of handle `H'` at `time(HT')` a new epoch `E'` is
+established when `H'` is received by each protocol participant.  The Attester
+relays the Attestation Result obtained during epoch `E` (associated with handle
+`H`) to the Relying Party using the handle for the current epoch `H'`.
```

Comment:
and what happens if the Relying Party hasn't gotten H' yet?   (I added a brief discussion of this in PR 257, so would be good to clarify here.  This section just has an example so any answer may be ok.)